packages/ui-filecoin/src/components/Markdown/MarkdownContent.tsx

```diff
@@ -16,6 +16,9 @@ export function MarkdownContent({ children }: MarkdownContentProps) {
     a: ({ href, ...props }) => (
       <MarkdownLink href={href as string} {...props} />
     ),
+    img: ({ src, alt }) => (
+      <img src={src as string} alt={alt as string} className="rounded-lg" />
+    ),
```

Comment:
What about opening a ticket to look into moving / reusing `MarkdownImage` to ui-filecoin?